tenant-isolation-config-service-api/src/main/proto/org/hypertrace/tenantisolation/config/service/v1/tenant_isolation_config_service.proto

```diff
@@ -0,0 +1,44 @@
+syntax = "proto3";
+
+package org.hypertrace.tenantisolation.config.service.v1;
+
+option java_multiple_files = true;
+
+import "org/hypertrace/tenantisolation/config/service/v1/tenant_isolation_group_config.proto";
+
+service TenantIsolationConfigService{
+  rpc GetAllTenantIsolationGroupConfigs(GetAllTenantIsolationGroupConfigsRequest) returns (GetAllTenantIsolationGroupConfigsResponse) {}
+  rpc CreateTenantIsolationGroupConfig(CreateTenantIsolationGroupConfigRequest) returns (CreateTenantIsolationGroupConfigResponse) {}
```

Comment:
In our previous conversations, you had mentioned that only read was needed at the moment. If that's still the case, can we drop the write side of this PR until and unless needed? If not, what's changed?

This PR is already far too big to meaningfully review -  and if we're not using write at the moment, we're just guessing as to the requirements there which often leads to breaking changes.

Reply:
We will need write too. Just that it won't be through any platform service but via a tool that SREs will use. For example if it is a cli it could still use the grpc APIs to update the config

Reply:
There's a use case for write, yes - my point was that it didn't need to be all in one giant first cut. The requirements you described for the original use case don't need write, and that would have greatly simplified the PR - that's why I thought we had discussed just populating this via helm config.